packages/core/src/scxml.ts

```diff
@@ -497,7 +491,18 @@ function scxmlToMachine(
   scxmlJson: XMLElement,
   options: ScxmlToMachineOptions
 ): MachineNode {
-  const machineElement = scxmlJson.elements!.find(
+  if (!scxmlJson.elements) {
+    // If there is no SCXML element, this should call 'done.invoke' immediately.
+    // TODO: there's probably a better way to do this
```

Comment:
Note: I think a couple SCXML tests literally have this same "finish immediately" machine  invoked in the tests 😝